com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/validators/parameters/v2/ModuleParametersCompatabilityValidatorTest.java

```diff
@@ -0,0 +1,86 @@
+package com.sap.cloud.lm.sl.cf.core.validators.parameters.v2;
+
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+import java.text.MessageFormat;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import com.sap.cloud.lm.sl.cf.core.model.SupportedParameters;
+import com.sap.cloud.lm.sl.cf.core.util.UserMessageLogger;
+import com.sap.cloud.lm.sl.mta.model.Module;
+
+public class ModuleParametersCompatabilityValidatorTest {
+
+    @Mock
+    private UserMessageLogger userMessageLogger;
+
+    @BeforeEach
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    public static Stream<Arguments> testModuleParametersCompatability() {
+        return Stream.of(
+        // @formatter:off
+            Arguments.of(Arrays.asList(SupportedParameters.HOST, SupportedParameters.ROUTES), true, Arrays.asList(SupportedParameters.HOST)),
```

Comment:
There is not case where routes, idle-routes and some not required params are defined.